* Add a define which will be toggled by configure when devd backend is activated
*
--- include/dix-config.h.in.orig	2020-03-30 21:48:11 UTC
+++ include/dix-config.h.in
@@ -400,6 +400,9 @@
 /* Support D-Bus */
 #undef HAVE_DBUS
 
+/* Use devd for input hotplug */
+#undef CONFIG_DEVD
+
 /* Use libudev for input hotplug */
 #undef CONFIG_UDEV
 
